src/components/Form/FormFields/TextFormField.tsx

```diff
@@ -101,6 +101,7 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
         name={field.name}
         value={field.value}
         required={field.required}
+        autoCapitalize={props.autoCapitalize || "words"}
```

Comment:
Not sure if enabling this for words by default for all other usages is a good idea.

```suggestion
        autoCapitalize={props.autoCapitalize}
```

Reply:
Oh just saw coderabbit's comment on the same... Let's remove the defaulting...

<img width="877" alt="image" src="https://github.com/user-attachments/assets/ba6a28cb-cbd8-475f-9e86-e0ab75f3068a">

Reply:
ok i will remove default "words"
but I am passing it as autoCaptalize = "words" in PatientRegister.tsx
<TextFormField
    required
    {...field("name")}
    type="text"
    label={"Name"}
    autoCapitalize="words"
 />
 
 this is fine right?